arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -2597,6 +2639,22 @@
       },
       "description": "Parameters of move resources."
     },
+    "ExportTemplateRequest": {
+      "properties": {
+        "resources": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the ids of the resources. The only supported string currently is '*' (all resources). Future api updates will support exporting specific resources."
+        },
+        "options": {
+          "type": "string",
+          "description": "The export template options. Supported values include 'IncludeParameterDefaultValue', 'IncludeComments' or 'IncludeParameterDefaultValue, IncludeComments"
+        }
```

Comment:
enum: [",", ...]
x-ms-enum: {
 name: ExportTemplateRequestOptions,
 modelAsString: false//if you want it as enum (validation happens), true // no validation (C# will have static class with static enum values. If you think there will be more values added in the future then keep it true.)